app/assets/javascripts/modules/WizardForm.js

```diff
@@ -0,0 +1,61 @@
+define(['jquery'], function($) {
+  'use strict';
+
+  var $form = $('[data-wizard-form] form'),
+      $formItems = $form.find('.wizard-form-item'),
+      $submitButton = $form.find('button.button--primary'),
+      $prevButton = $form.find('button.button--secondary'),
+      $currentStep = $formItems.first(),
+      $currentStepNumber = 0;
+
+  $formItems.hide().first().show().addClass('show');
```

Comment:
Our preference is to keep the hidden/shown states of these items in the CSS by adding classes to the element rather than directly setting their display in the JS.

Reply:
Agreed, I moved initial showing/hiding in css now